packaging/os/yumrepo.py

```diff
@@ -134,6 +174,14 @@
         are allowed. Substitution variables (e.g. C($releasever)) are honored
         here.
       - The list can also be a regular YAML array.
+  ip_resolve:
```

Comment:
While "whatever" is [official](http://lists.baseurl.org/pipermail/yum-devel/2010-April/006900.html), it's a [silly](https://www.mail-archive.com/yum-devel@lists.baseurl.org/msg01500.html) name. Would it make sense to divert from the official man page and specify that "whatever" is indeed a legitimate option and not "foobar"?